benchmarks/benchmark.py

```diff
@@ -444,6 +448,7 @@ def modify_config_props_for_mms(pargs):
     target = parser.add_mutually_exclusive_group(required=True)
     target.add_argument('name', nargs='?', type=str, choices=benchmark_name_options, help='The name of the benchmark to run')
     target.add_argument('-a', '--all', action='store_true', help='Run all benchmarks')
+    target.add_argument('--model-name', nargs=1, type=str, default=[MODEL_RESNET_18], choices=MODEL_MAP.keys(), help='A preloaded model to run.  It defaults to {}. Use this argument to run all the benchmarks on this single model'.format(MODEL_RESNET_18))
```

Comment:
Having this accept a model means there are two ways to pass in the model.  It is clearer to instead make this argument take a boolean similar to --all that simply changes the mode, and then reuse the --model/--custom-model parameter.  This better matches the behavior of name and all, as well as enables running the set of tests on custom models.  Maybe name running the set of tests for a given model "--suite"?

Reply:
Good point. 
Done.